Gordon360/Services/NewsService.cs

```diff
@@ -306,6 +327,27 @@ private static bool VerifyUnapproved(StudentNews newsItem)
             return true;
         }
 
+        /// <summary>
+        /// Helper method to verify that a given news item has already been approved
+        /// </summary>
+        /// <param name="newsItem">The news item to verify</param>
+        /// <returns>true ig approved, otherwise throws some kind of meaningful exception</returns>
+        private static bool VerfiyApproved(StudentNews newsItem)
+        {
+            // Note: These checks have been duplicated from StateYourBusiness because we do not want
+            //    SuperAdmins to be able to delete expired news, this should be fixed eventually by
+            //    removing some of the SuperAdmin permissions that are not explicitly given
+            if (newsItem.Accepted == null)
+            {
+                throw new ResourceNotFoundException() { ExceptionMessage = "The news item acceptance status could not be verified." };
+            }
+            if (newsItem.Accepted == false)
+            {
+                throw new BadInputException() { ExceptionMessage = "The news item has not been approved." };
+            }
+            return true;
+        }
```

Comment:
These two checks are really for the same edge case, where the news item is not yet approved. You could condense this down to:
```suggestion
            if (newsItem.Accepted != true)
            {
                throw new BadInputException() { ExceptionMessage = "The news item has not been approved." };
            }
            return true;
        }
```